### DISCUSSION TITLE
Post-install Configuration, are there any docs?

### DISCUSSION THREAD

**Discussion by ag-gaphp** [OpnForm/OpnForm#480] [Q&A]
The only docs I was able to find are in the README, but it doesn't mention anything about post-install setup and configuration.

I mainly want to disable account sign-ups, **dis-allow form creation without an account**, and setup an administrator account to manage users, forms, etc. from the frontend. Is any of that possible? Most applications have a default admin account, or allow you to create the admin account on installation. I don't see that information documented anywhere in the README.

I'm not even sure if an administrator type account exists in this system, and if it does what does it have the ability to do. I'm hoping I have just overlooked the official documentation.

**Comment by Dvalin21** [comment]
Agreed +1

**Comment by Dvalin21** [comment]
I found this
https://discord.gg/YTSjU2a9TS

**Reply by ag-gaphp** [reply]
Thanks, but some folks can't use Discord, unfortunately. I always skip over it cause it's blocked for me at work and I'm not making an account at home just to get documentation for this.

**Comment by JhumanJ** [comment]
Hi guys - we will make this the default behavior very soon (we have pending PRs). We're just migrating to a more solid docker setup, and we'll deliver very soon 🙌

**Reply by ag-gaphp** [reply]
Thanks, but what does the Docker setup have to do with post-install configuration documentation?

Edit: I also don't see any PRs for documentation

**Comment by ag-gaphp** [comment]
I don't understand why this was closed. The response from the dev talks about "default behavior", but I'm not asking for a new feature or for a bug to be fixed. I'm asking for **documentation on how to configure this app**, which is not behavior of the app itself but information on how to use it as a sysadmin/dev.

**Comment by JhumanJ** [comment]
Hi @ag-gaphp you're right this has nothing to do directly with the docker setup, but we're working on these subjects step by step! Please wait for a week or so 🙏
